yagi/src/main/java/org/raml/yagi/framework/nodes/snakeyaml/SYPosition.java

```diff
@@ -73,6 +75,19 @@ public void setIncludedResourceUri(String includedResourceURI)
         this.includedResourceURI = includedResourceURI;
     }
 
+    @Nullable
+    @Override
+    public String getIncludedResourcePath()
+    {
+        return includedResourcePath;
+    }
+
+    @Override
+    public void setIncludedResourcePath(String includedResourcePath)
+    {
+        this.includedResourcePath = includedResourcePath;
+    }
+
```

Comment:
This code is duplicated here and in `DefaultPosition`, maybe you can add it in AbstractPosition so both classes will inherit this code